recipes/synthpop/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: synthpop
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/synthpop/synthpop-{{ version }}.tar.gz
+  sha256: dfbea2123a00801d30bc62e47f6d977a91424022227d2747527564f35c3a5805
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - census>=0.5
```

You need a single space here or the feedstock creation will fail
```suggestion
    - census >=0.5
```
(same for the others)

---

Ooh, thanks for catching this! Just fixed it